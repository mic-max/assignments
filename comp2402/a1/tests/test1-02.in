Hello
hello
Cheese
cheese
Potato
potato
car
Car