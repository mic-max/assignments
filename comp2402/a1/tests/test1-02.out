Car
car
potato
Potato
cheese
Cheese
hello
Hello
